beacon-chain/node/node.go

```diff
@@ -435,11 +436,16 @@ func (b *BeaconNode) registerPOWChainService() error {
 			"You will need to specify --http-web3provider to attach an eth1 node to the prysm node. Without an eth1 node block proposals for your validator will be affected and the beacon node will not be able to initialize the genesis state.",
 		)
 	}
-	endpoints := []string{b.cliCtx.String(flags.HTTPWeb3ProviderFlag.Name)}
```

Comment:
why not just provide raw strings to the powchain service ? and then do the parsing there itself. You dont need to parse it here.